tests/foreman/destructive/test_contenthost.py

```diff
@@ -30,14 +30,13 @@ def test_content_access_after_stopped_foreman(target_sat, rhel7_contenthost):
 
     :CaseImportance: Medium
 
-    :CaseComponent: Infrastructure
+    :CaseComponent: Hosts-Content
 
-    :Team: Platform
+    :Team: Phoenix-subscriptions
```

Comment:
@jameerpathan111 Do you intend to cherrypick this assignment separately for zstream branches?